src/cascadia/TerminalApp/App.cpp

```diff
@@ -159,13 +162,75 @@ namespace winrt::TerminalApp::implementation
         {
             IInspectable g = res.Lookup(key);
             winrt::Windows::UI::Xaml::Style style = g.try_as<winrt::Windows::UI::Xaml::Style>();
+            _root.Style(style);
             _tabRow.Style(style);
         }
 
         // Apply the UI theme from our settings to our UI elements
         _ApplyTheme(_settings->GlobalSettings().GetRequestedTheme());
 
         _OpenNewTab(std::nullopt);
+
+        _root.Loaded({ this, &App::_OnLoaded });
+    }
+
+    // Method Description:
+    // - Show a ContentDialog with a single "Ok" button to dismiss. Looks up the
+    //   the title and text from our Resources using the provided keys.
+    // - Only one dialog can be visible at a time. If another dialog is visible
+    //   when this is called, nothing happens.
+    // Arguments:
+    // - contentKey: The key to use to lookup the title text from our resources.
+    // - textKey: The key to use to lookup the content text from our resources.
+    fire_and_forget App::_ShowOkDialog(const winrt::hstring& contentKey,

```

Comment:
per #721, this method should be renamed to `_showOkDialog`

Reply:
I am also technically OK keeping the bad capitalization here and using #721 as a bulk cut-over point +/- automatic enforcement.

Reply:
I think I'm going to resolve all the capitalization comments as a wontfix - lets just use 721 as an opportunity to do that